### DISCUSSION TITLE
[Feature] Disable backup on mobile data

### DISCUSSION THREAD

**Discussion by anevaiopvn** [immich-app/immich#11904] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to have an option to disable backups while on mobile data (iOS). I have a limited data plan so this is a must with large videos. I’m thinking I must have missed something because this seems like such an obvious feature. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
In the app's backup screen, press the cog on the top right to find the wifi-only setting.

**Reply by anevaiopvn** [reply]
Okay that’s good enough. That only applies to auto background sync which I didn’t have enabled, and not “automatic backups” which I now turned off. 

Well it’s a solution nonetheless. Thx.
